### PR TITLE
Fixes the universal scanner's sprites and adds contextual screentips.

### LINKED ISSUE #70909: Sales tagger doesnt work, also "QOL" change actually harshly affected the accessibility of custom vendors
Reporting client version: 514.1582

## Testmerges:
- [Fix halloweens races](https://www.github.com/tgstation/tgstation/pull/70874)

## Reproduction:
Try to print sales tagger for my custom vendor shop
Cant print it from autolathe
Look up and read that #70314 merged it with export scanner as some "QOL" pr 
Break into cargo
Cant print from cargo printer
Have to get the universal scanner from QM locker (LIMITED AVAILABILITY?!??!?!) (Edit:) melbert pointed out you can also buy up to 3 from the youtool vendor for 95 credits each
Set it to sales function
I can only set the price of items to 1, no matter what number i input making my custom vendor pretty much useless.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This PR fixes a minor icon_state name discrepancy when switching to the sales tagger mode (I forgot to add a space to the icon_state, which carried over to the radial).

I then read #70909 which brings up several issues with the item, and after actually testing it, everything works as intended. The issue it appears is that it's not clear enough that the functionality is the same as it was previously on the prior items, so we've added contextual screentips to all 3 modes of the scanner.

image

## Why It's Good For The Game

Fixes #70909. Improves player feedback on the three modes of the universal scanner through context.

## Changelog

:cl:
fix: The universal scanner now properly show all three options in the radial with sprites, due to a missing space.
qol: The universal scanner now has contextual screentips within all three of its modes.
/:cl:

### CODE REVIEW

File: code/modules/cargo/universal_scanner.dm [hunk 1]
```diff
@@ -145,8 +146,24 @@
 			. += span_notice("<b>Ctrl-click</b> to clear the registered account.")
 
 	if(scanning_mode == SCAN_PRICE_TAG)
-		. += span_notice("The current custom price is set to [new_custom_price] cr.")
+		. += span_notice("The current custom price is set to [new_custom_price] cr. <b>Right-Click</b> to change.")
 
+/obj/item/universal_scanner/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+	//if (held_item != src)
+	//	return NONE
```
Comment (ShizCalev) [inline code/modules/cargo/universal_scanner.dm:154]: Can remove this if you're not using it.

File: code/modules/cargo/universal_scanner.dm [hunk 2]
```diff
@@ -145,8 +146,24 @@
 			. += span_notice("<b>Ctrl-click</b> to clear the registered account.")
 
 	if(scanning_mode == SCAN_PRICE_TAG)
-		. += span_notice("The current custom price is set to [new_custom_price] cr.")
+		. += span_notice("The current custom price is set to [new_custom_price] cr. <b>Right-Click</b> to change.")
 
+/obj/item/universal_scanner/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
```
Comment (ShizCalev) [inline code/modules/cargo/universal_scanner.dm:152]: Don't need `. = ..()`

**Comment by TetrakoHateThis** [comment]
We noticed an issue downstream that the Universal Scanner lacks recipes for printing out on lathes (auto and cargo). No idea if that's also an issue up here or not but figured I'd post about it.

File: code/modules/cargo/universal_scanner.dm [hunk 3]
```diff
@@ -145,8 +146,24 @@
 			. += span_notice("<b>Ctrl-click</b> to clear the registered account.")
 
 	if(scanning_mode == SCAN_PRICE_TAG)
-		. += span_notice("The current custom price is set to [new_custom_price] cr.")
+		. += span_notice("The current custom price is set to [new_custom_price] cr. <b>Right-Click</b> to change.")
```
Comment (Mothblocks) [inline code/modules/cargo/universal_scanner.dm:149]:
```suggestion
		. += span_notice("The current custom price is set to [new_custom_price] cr. <b>Right-click</b> to change.")
```

File: code/modules/cargo/universal_scanner.dm [hunk 4]
```diff
@@ -117,7 +118,7 @@
 		if(!chosen_price || QDELETED(user) || QDELETED(src) || !user.canUseTopic(src, be_close = TRUE, no_dexterity = FALSE, no_tk = TRUE) || loc != user)
 			return
 		new_custom_price = chosen_price
-		to_chat(user, span_notice("The [src] will now give things a [new_custom_price] cr tag."))
+		to_chat(user, span_notice("\The [src] will now give things a [new_custom_price] cr tag."))
```
Comment (Mothblocks) [inline code/modules/cargo/universal_scanner.dm:121]:
```suggestion
		to_chat(user, span_notice("[src] will now give things a [new_custom_price] cr tag."))
```

"The" is implicit.

File: code/modules/cargo/universal_scanner.dm (hunk 1 above)
Comment (Mothblocks) [inline code/modules/cargo/universal_scanner.dm:154]:
```suggestion

```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/cargo/universal_scanner.dm (hunk 2 above)
Comment (Mothblocks) [reply code/modules/cargo/universal_scanner.dm:152]:
```suggestion
```

**Pull request merged**